scripts/common.py

```diff
@@ -214,7 +214,10 @@
 ACCEPTANCE_TESTS_SUITE_NAMES = [
     'blog-admin-tests/assign-roles-to-users-and-change-tag-properties.spec.js',
     'blog-editor-tests/check-blog-editor-unable-to-publish-' +
-    'duplicate-blog-post.spec.js'
+    'duplicate-blog-post.spec.js',
+    'translation-admin-tests/provide-translation-rights.spec.js',
```

Comment:
also add the new tests in the github action workflows for the acceptance tests: https://github.com/oppia/oppia/blob/c6712344e62bfc3ca53bf0a64142336bd92ab706/.github/workflows/e2e_lighthouse_performance_acceptance_tests.yml#L291-L293

Reply:
Done.